### PR TITLE
mpu6050: add i2c aux bypass

### PR DESCRIPTION
**Pull request by ric96**
Signed-off-by: Sahaj Sarup <sahaj.sarup@linaro.org>

### CODE REVIEW

**Review by MaureenHelm — APPROVED** [review]

File: drivers/sensor/mpu6050/Kconfig [hunk 1]
```diff
@@ -65,4 +65,7 @@ config MPU6050_GYRO_FS
 	  An X value for the config represents a range of +/- X degrees/second.
 	  Valid values are 250, 500, 1000, 2000.
 
+config MPU6050_I2C_BYPASS
+	bool "Bypasss auxiliary i2c bus to main i2c bus."
```
Comment (avisconti) [inline drivers/sensor/mpu6050/Kconfig:69]: default is true, correct? Is that what you want?

**Review by avisconti — CHANGES_REQUESTED** [review]
Please correct typos in the commit message

File: drivers/sensor/mpu6050/Kconfig (hunk 1 above)
Comment (avisconti) [reply drivers/sensor/mpu6050/Kconfig:69]: Sorry is "n", so I guess this is ok

**Comment by avisconti** [comment]
@ric96 Can you change the commit message? I'm saying this again just to avoid confusion about my review.

**Pull request closed without merging**